app/database/models/task_comment.py

```diff
@@ -0,0 +1,93 @@
+from app.database.sqlalchemy_extension import db
+from datetime import datetime
+
+
+class TaskCommentModel(db.Model):
+    """Data Model representation of a task comment.
+
+    Attributes:
+        id: integer primary key that defines the task comment.
+        user_id: integer indicates id of user.
+        task_id: integer indicates id of task.
+        creation_date: float that defines the date of creation of the task comment.
+        modification_date: float that defines the latest date of modification of the task comment.
+        comment: string that defines the comment of task comment.
+    """
+
+    # Specifying database table used for Task comment model
+    __tablename__ = 'task_comments'
+    __table_args__ = {'extend_existing': True}
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # personal data
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    task_id = db.Column(db.Integer, db.ForeignKey('tasks_list.id'))
+    creation_date = db.Column(db.Float, nullable=False)
+    modification_date = db.Column(db.Float)
+    comment = db.Column(db.String(500), nullable=False)
+
+    def __init__(self, user_id, task_id, comment):
+        self.user_id = user_id
+        self.task_id = task_id
+        self.comment = comment
+
+        self.creation_date = datetime.now().timestamp()
+
+    def json(self):
+        """Returns information of task comment as a json object."""
+        return {
+            'id': self.id,
+            'user_id': self.user_id,
+            'task_id': self.mentor_id,
+            'creation_date': self.creation_date,
+            'modification_date': self.modification_date,
+            'comment': self.comment
+        }
+
+    @classmethod
+    def find_by_id(cls, _id):
+        """Returns the task comment that has the passed id.
+           Args:
+                _id: The id of a task comment.
+        """
+        return cls.query.filter_by(id=_id).first()
+
+    @classmethod
+    def find_by_user_id(cls, user_id):
```

Comment:
I don't see a valid use case for this method, can you?

Reply:
find_by_user_id can be used to retrieve all task comments made by that user across all tasks. This could be useful if that user ever wants to refer to a comment he/she made in a past relationship etc.